### DISCUSSION TITLE
PrusaSlicer FILEFORMAT_V4.CTB does not convert automatically anymore?

### DISCUSSION THREAD

**Discussion by msspad** [sn4k3/UVtools#419] [Q&A] [answered]
I have used the following configuration in PrusaSlicer Notes:
```
PRINTER_VENDOR_PRUSA3D
PRINTER_MODEL_SL1
PRINTER_VENDOR_PHROZEN
PRINTER_MODEL_SONIC_MINI_4K
FILEFORMAT_V4.CTB
```
and activated Settings -> Automations -> Auto convert SL1. It used to convert to v4.ctb files but now, no conversion happens anymore. How can I convert SL1 automatically to v4.ctb (for Phrozen Mini 4k)?

Also, can you give a minHeight to the properties of the file? Otherwise, "Layer data" hides the file properties sometimes completely:
![image](https://user-images.githubusercontent.com/6191764/154449081-7e17df2e-c89a-49a4-8fdf-73228855e5a0.png)

**Comment by sn4k3** [comment, marked as answer]
**Vn.ext** was drop to simplify and in favour of the new standard of setting versions

```
PRINTER_VENDOR_PRUSA3D
PRINTER_MODEL_SL1
PRINTER_VENDOR_PHROZEN
PRINTER_MODEL_SONIC_MINI_4K
FILEFORMAT_CTB
FILEVERSION_4
```

> Also, can you give a minHeight to the properties of the file? Otherwise, "Layer data" hides the file properties sometimes completely

It looks like a good thing to do in theory, but in pratice the UI framework is still a mess when it come to auto sizes and limits :(
See what happens when i set MinHeight=200 to Properties:

![UVtools_2022-02-17_15-27-56](https://user-images.githubusercontent.com/113281/154514608-50d0f7bc-ae29-46d0-9036-c6ab4b2d8e43.png)

The layer data dont even scale! And even if you force, no scroll bars are shown. I'm stick of report those auto size bugs in framework but i never get an anwser.

All i can do is add a spliter to be able to resize manually when you want, manual size is respected well.

**Reply by msspad** [reply]
FILEVERSION_4 works perfectly, thanks!

It's a pity that the framework does not solve resizing properly. A manual handle for resizing would certainly help. Not sure it's a good idea but maybe an option to hide the preview images could also help? Or move the images to an other tab?

**Reply by sn4k3** [reply]
> It's a pity that the framework does not solve resizing properly. A manual handle for resizing would certainly help. Not sure it's a good idea but maybe an option to hide the preview images could also help? Or move the images to an other tab?

I could handle that on code, but it's a pain when that should be part of framework job.
You can already "hide" the preview, it have a splitter between image and properties, you can drag it and push up or down

**Reply by msspad** [reply]
ah, I didn't notice the splitter before, thanks. Will also use it.

Another minor detail I noticed in ScriptDebandingZSample.cs: I think you removed once firstLayer.SetNoDelays(); what results in waiting for WaitTimeBeforeCure=40s for nothing.

**Reply by sn4k3** [reply]
> Another minor detail I noticed in ScriptDebandingZSample.cs: I think you removed once firstLayer.SetNoDelays(); what results in waiting for WaitTimeBeforeCure=40s for nothing.

I dont renember why i commented that line, but i can activate that again
